lib/protocol/message.go

```diff
@@ -126,14 +127,23 @@ func (o *ClusterConfigMessage) GetOption(key string) string {
 }
 
 type Folder struct {
-	ID      string   // max:256
-	Devices []Device // max:1000000
-	Flags   uint32
-	Options []Option // max:64
+	ID              string // max:256
+	ReadOnly        bool
+	RescanIntervalS int
```

Comment:
We should define this explicitly as int32 or int64 for the XDR encoding to be unambiguous.